client/my-sites/github-deployments/components/page-shell/index.tsx

```diff
@@ -14,6 +23,44 @@ interface GitHubDeploymentsProps {
 }
 
 export function PageShell( { topRightButton, pageTitle, children }: GitHubDeploymentsProps ) {
+	const siteId = useSelector( getSelectedSiteId );
+	const isSiteAtomic = useSelector( ( state ) => isSiteWpcomAtomic( state, siteId as number ) );
+	const dispatch = useDispatch();
+	const transferState = useSelector( ( state ) => getAutomatedTransferStatus( state, siteId ) );
+	const [ hasTransfer, setHasTransferring ] = useState(
+		!! (
+			transferState &&
+			transferInProgress.includes( transferState as ( typeof transferInProgress )[ number ] )
+		)
+	);
+	const showHostingActivationBanner = ! isSiteAtomic && ! hasTransfer;
+
+	const clickActivate = () => {
+		dispatch( initiateThemeTransfer( siteId ?? 0, null, '', '', 'hosting' ) );
+		setHasTransferring( true );
+	};
+
+	const getAtomicActivationNotice = () => {
+		return (
+			<Notice
+				className="hosting__activating-notice"
+				status="is-info"
+				showDismiss={ false }
+				text={ translate( 'Please activate the hosting access to begin using these features.' ) }
```

Comment:
I know you've taken this string from the existing section which saves some translation, but without reference to "hosting access" on the screen, it makes me think "what hosting access?" - thoughts?! 

```suggestion
				text={ translate( 'Please activate hosting access to begin using this feature.' ) }
```

Also more in-line with strings like this:

https://github.com/Automattic/wp-calypso/blob/75113ecfc48c8e415791ad4366a867bd9a36ce6c/client/blocks/eligibility-warnings/warning-list.tsx#L110

Just a minor proposal - feel free to ignore. :)

Reply:
Good catch @Aurorum

Reply:
@Aurorum do you think we should redirect to the eligibilty warning component similar to `/hositng-config`? I was trying to avoid that step.

Reply:
No, sorry - I don't think that step is necessary either! Was just using it as an example of a string where "hosting access" is used over "the hosting access". :)